src/app/parcels/form/formSections/PackingSlotsCard.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { CardProps, errorText, valueOnChangeDropdownList } from "@/components/Form/formFunctions";
+import GenericFormCard from "@/components/Form/GenericFormCard";
+import { ErrorText } from "@/components/Form/formStyling";
+import DropdownListInput from "@/components/DataInput/DropdownListInput";
+import { PackingSlotsLabelsAndValues } from "@/common/fetch";
+
+interface PackingSlotsCardProps extends CardProps {
+    packingSlotsLabelsAndValues: PackingSlotsLabelsAndValues;
+}
```

Comment:
nitpick: empty line for separation
```suggestion
}

```